Network chat

VladimirSinitsyn opened the pull request:
Part 1

Adding method for sending message to necessary participant

VladimirSinitsyn committed:
Network chat
Part 1

Adding method for sending message to necessary participant

OzzyFrost commented on server/src/main/java/network_chat/server/ClientHandler.java:
```diff
@@ -0,0 +1,103 @@
+                                sendMessage("Ошибка: Неверный формат команды. Используйте /w <ник> <сообщение>");
+                                continue;
+                            }
+                        }
```
эту фигурную скобку надо переместить на 53 строку. 
ведь с 37 по 53 строки это все обработка приватных сообщений

VladimirSinitsyn replied:
Перенес, спасибо. Помню, были с этой скобкой проблемы и даже помню, что переносил, но видимо потом что-то пошло не так в момент отладки. В этому месте много времени потратил.

VladimirSinitsyn committed:
Network chat
Part 1

Adding method for sending message to necessary participant

Updated ClientHandler's code. } moved from 42 to 53 line
